planus/src/lib.rs

```diff
@@ -30,9 +30,9 @@ pub use crate::{
 #[doc(hidden)]
 pub const fn check_version_compatibility(s: &str) {
     match s.as_bytes() {
-        b"planus-0.3.0" => (),
+        b"planus-0.3.1" => (),
```

Comment:
At some point we should probably consider our semantic versioning guarantees with regards to this code.